types/index.d.ts

```diff
@@ -62,6 +62,7 @@ export const Switch: FunctionComponent<SwitchProps>;
 
 export interface RouterProps {
   hook: LocationHook;
+  basepath: string;
```

Comment:
While it's not really critical, but I would suggest using a `Path` type here (which is just an alias to `string`, but helps to keep the code more generic I guess). Basically `Path` represents something that can describe a location.

Reply:
good idea!! Thx! i will fix it